pkg/cmd/config.go

```diff
@@ -425,7 +425,8 @@ clusterNetwork:
   cni:
     # possible values:
     # * canal
-    # * weave-net
+	# * weave-net
+	# * external - The CNI plugin can be installed as Addon or manually
```

Comment:
```suggestion
    # * weave-net
    # * external - The CNI plugin can be installed as Addon or manually
```
Tiny nit: wrong indentation, please use spaces instead of tabs

Reply:
fixed